pkgs/development/tools/wizer/default.nix

```
@@ -0,0 +1,31 @@
+{ fetchCrate, lib, rustPlatform }:
```

Minor style nit, we usually put `lib` first though. (I switched the order of `fetchCrate` and `rustPlatform`, since the latter is used first. Feel free to not make that change, if you'd like -- it doesn't matter.)

```suggestion
{ lib, rustPlatform, fetchCrate }:
```

---

Works for me, and glad to be aware of this best practice!